docs/guides/get-started/aws-credentials.md

```diff
@@ -54,8 +54,8 @@ Lastly, you will receive a Success message where you will see your **Access key
 
 ![aws-user-created](/img/guides/aws-iam/aws-user-created.png)
 
-Now that we have the **Access key ID** and **secret access key** we can put them in `~/.aws/credentials`. It should look something like this:
+Now that we have the **Access key ID** and **secret access key** we can put them in `~/.aws/credentials` on Linux/macOS or at `C:\Users\USERNAME.aws\credentials`on Windows.It should look something like this:

```

Comment:
The windows path, I believe is missing a slash after the username. Please double check.

Reply:
Modified